### PR TITLE
docs: Update custom.scss with style previously applied from github.css

### PR DESCRIPTION
**Pull request by JamilAhmad**
I have updated the custom.scss file according to the theme documentation and implemented the new zeplin design changes properly hope things will be good now, please review and let me know if any question

### CODE REVIEW

File: docs/_sass/custom/custom.scss [hunk 1]
```diff
@@ -799,3 +867,21 @@ th, td {
 .tabs .ui-tabs-nav .ui-state-active a {
 	color: $teal-green;
 }
+/*overridden*/
```
Comment (nopcoder) [inline docs/_sass/custom/custom.scss:870]: The next part doesn't look like scss - example I found: https://gist.github.com/akullpp/6375168d0400ee2e7e3a93820d94ba54

**Review by nopcoder — CHANGES_REQUESTED** [review]
wrote a couple of comments in the pull request body.
as part of customizing the default theme, I would try to reduce the use of  `!important` as must as possible.

**Review by nopcoder — CHANGES_REQUESTED** [review]
1. can you address the open question I left as comment?
2. please remove the overrides we pushed into the github.css and see that these changes apply without them?

File: docs/_sass/custom/custom.scss [hunk 2]
```diff
@@ -15,6 +15,11 @@
 	src: local(GalanoGrotesque-Bold),
 		url('../fonts/Rene Bieder - Galano Grotesque Bold.otf');
 }
+@font-face {
+	font-family: Rene-Bieder-Galano-Grotesque-Medium;
+	src: local(Rene-Bieder-Galano-Grotesque-Medium),
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Fixed name of the font and changed it to Galano-Grotesque-Medium

File: docs/_sass/custom/custom.scss [hunk 3]
```diff
@@ -33,6 +38,13 @@ $white-two: #f8f8f8;
 $charcoal: #2f3433;
 $note: #fff3e5;
 $note-border: #f3a342;
+$whitegrey:#f5f7f9;
+$lightblack:#6e7171;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Removed All repetition

File: docs/_sass/custom/custom.scss [hunk 4]
```diff
@@ -33,6 +38,13 @@ $white-two: #f8f8f8;
...
+$darkgrey:#949494;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Removed All repetition

File: docs/_sass/custom/custom.scss [hunk 5]
```diff
@@ -63,51 +75,98 @@ body {
 			max-width: 25%;
 			position: relative;
 			height: auto;
-			background-color: $white-two;
+			background-color: $whitegrey;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss:77]: Removed one, now its only one background property

File: docs/_sass/custom/custom.scss [hunk 6]
```diff
@@ -63,51 +75,98 @@ body {
...
+		font-family: Rene-Bieder-Galano-Grotesque-Medium;
+	    font-size: 14px;
+	}
 
 	h1 {
 		margin-top: 40px !important;
-		font-family: GalanoGrotesque;
-		font-size: 42px;
+		font-family: Rene-Bieder-Galano-Grotesque-Medium;
+		font-size: 42px!important;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Fixed formate

File: docs/_sass/custom/custom.scss [hunk 7]
```diff
@@ -63,51 +75,98 @@ body {
...
+			font-family: Rene-Bieder-Galano-Grotesque-Medium;
+			color: $white;
+			font-size: 14px !important;
 		}
 	}
 
 	h3 {
 		margin-top: 25px !important;
-		font-size: 22px;
+		font-size: 18px!important;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Removed important where not needed

File: docs/_sass/custom/custom.scss [hunk 8]
```diff
@@ -63,51 +75,98 @@ body {
...
 			&::before {
 				display: none;
 			}
 			a {
-				color: $black;
-				font-size: 16px;
-				padding: 8px 0;
+				color: $teal-green;
+				font-size: 14px;
+				/* padding: 8px 0; */
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Comments removed

File: docs/_sass/custom/custom.scss [hunk 9]
```diff
@@ -126,7 +185,9 @@ body {
 	}
 
 	pre code {
-		background-color: initial;
+		/* background-color: initial; */
+		background-color: #575e65;
+    	border: 0px solid #d8faf1;
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Fixed

File: docs/_sass/custom/custom.scss [hunk 10]
```diff
@@ -126,7 +185,9 @@ body {
 	}
 
 	pre code {
-		background-color: initial;
+		/* background-color: initial; */
```
[the remark this answers is not in the document]
Comment (JamilAhmad) [reply docs/_sass/custom/custom.scss]: Removed

**Review by JamilAhmad — COMMENTED** [review]
All done

**Comment by nopcoder** [comment]
> All done

please remove the overrides we pushed into the github.css and see that these changes apply without them?

**Review by nopcoder — APPROVED** [review]

**Pull request merged**